### PR TITLE
Add parliament scraping to the scraper

### LINKED ISSUE #98: Scrape the Parliament website

### PR DESCRIPTION
**Pull request by SamDepardieu**
# Description

This PR aims to let us scrape the parliament website to find Wellcome references into debates, commitees and other documents. To achieve this goal, this PR:
- Adds a `parliament_spider` (`scrapy crawl parliament`)
- Fix a couple of bugs spotted dureing the developement
- Update the related documentation

Fix #98 

## Type of change

Please delete options that are not relevant.

- [x] :sparkles: New feature
- [x] :memo: Documentation update

# How Has This Been Tested?

Local runs only. We should stop including spider contracts and find a better way to test spiders.

# Checklist:

- [x] My code follows the style guidelines of this project (pep8 AND pyflakes)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] If needed, I changed related parts of the documentation
- [ ] I included tests in my PR [Not relevant]
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] If my PR aims to fix an issue, I referenced it using `#(issue)`

### CODE REVIEW

File: tools/dbTools.py [hunk 1]
```diff
@@ -34,7 +34,9 @@ def __del__(self):
         except AttributeError:
             if hasattr(self, 'logger'):
                 # __del__ can be called before self.logger is created.
-                self.logger.warning('Tried to close a non-existent connection.')
+                self.logger.warning(
+                    'Tried to close a non-existent connection.'
+                )
```
Comment (hblanks) [inline tools/dbTools.py]: Ah, yes, our style must be 72 chars. Sorry, will update the vimrc.

Comment (hblanks) [reply tools/dbTools.py]: One last thing we could do is add the command used to push the docker image to AWS into the Makefile.

Comment (hblanks) [reply tools/dbTools.py]: (but only if that's easy)

Comment (SamDepardieu) [reply tools/dbTools.py]: The docker push requires the output from `aws login`, so I don't know how we could do that in an easy, clean way at the moment. Maybe for a separate PR?

Comment (hblanks) [reply tools/dbTools.py]: Methinks everything we need could be set ahead of time in the environment...but I'll chat with you. Good to merge this meantime, one monkey don't stop the show :-)

Comment (SamDepardieu) [reply tools/dbTools.py]: Um, I think I see what you mean by settings things ahead, but still good to have a chat to be sure 👍

**Pull request merged**